counter/src/pages/FormPage.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import { Box, TextField, Button, Stack } from "@mui/material";
+import { DatePicker, LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { TextareaAutosize } from "@mui/base/TextareaAutosize";
+import Grid from "@mui/material/Grid";
+import TablePage from "./TablePage";
+import { addList, arr } from "./Functions";
```

Comment:
remove this component . manage this in the same component in state.

Reply:
Removed sir.